dashboard/app/models/pd/enrollment_constants.rb

```diff
@@ -0,0 +1,5 @@
+module Pd
+  module EnrollmentConstants
+    NO_NEW_FACILITATOR = "No (New Facilitator)"
```

Comment:
Suggest `SCHOLARSHIP_INELIGIBLE_NEW_FACILITATOR`. We will have constants with the same `SCHOLARSHIP_INELIGIBLE_` prefix when we add more reasons to deny scholarship.